Module/zipper.py

```diff
@@ -1575,3 +1579,127 @@ def _should_run_khbr():
 
             data.seek(0)
             return data
+
+class CompanionZip:
+    def __init__(self, seed_name: str, ui_settings: SeedSettings):
+        self.settings = ui_settings
+        self.donald_changed = False
+        self.goofy_changed = False
+        if self.settings.get("donald_damage_toggle") == True:
```

Comment:
When I was using my own atkp_list file, I noticed that my changes were getting overwritten unless I made them highest priority (might be goa upon reflection). I was considering this though, it might be fine if I included this within the rando seed zip file itself. Ultimately, I spent most of my time focusing on getting the yml file written correctly and dealing with creating the atkp related stuff in openkhmod.py, so making it write to the rando seed zip file instead shouldn't be too hard I think?